src/lib/transaction/Signatures.svelte

```diff
@@ -1,18 +1,21 @@
 <script lang="ts">
     import { StrKey, xdr } from 'stellar-sdk';
-
+    import WalletLanguage from '../../helpers/WalletLanguage';
     export let signatures: xdr.DecoratedSignature[];
+
+    const language = new WalletLanguage();
+    const lang = language.getText();
```

Comment:
I'm thinking this should exist in a store instead of existing in each component. Thoughts?